sysext/sysext.go

```diff
@@ -0,0 +1,95 @@
+package sysext
+
+import (
+	"archive/tar"
+	"fmt"
+	"github.com/kairos-io/kairos-sdk/types"
+	"io"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+)
+
+// DefaultAllowListRegex provided for easy use of defaults for confext and sysext
+var DefaultAllowListRegex = regexp.MustCompile(`^usr|^/usr|^etc|^/etc`)
```

Comment:
what if the dir is called `etcetera`? maybe it needs to have `^etc/` and/or `^etc$` and not just `^etc` there? I know it's not likely that a user will add that but 🤷

Reply:
Umm yes, very good point, I'll fix it!